SecureConfig/build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:4.0.2'
```

Comment:
4.1 is already out https://developer.android.com/studio/releases/gradle-plugin#4-1-0

Reply:
Done.